tests/test_conflicting_formatters.py

```diff
@@ -0,0 +1,72 @@
+import tempfile
+from typing import Generator, List, Type
+
+import pytest
+from _pytest.fixtures import SubRequest
+
+from pydocstringformatter import _testutils as test_utils
+from pydocstringformatter._utils.exceptions import UnstableResultError
+from pydocstringformatter.run import _Run
+
+
+@pytest.fixture(scope="function")
+def patched_run(request: SubRequest) -> Generator[Type[_Run], None, None]:
+    """Patches the _Run class to use the MakeAFormatter and MakeBFormatter."""
+    # This monkeypatches the formatters to make them conflict.
+    # and the only formatters applied
+    # pylint: disable=wrong-import-position, import-outside-toplevel
+    from pydocstringformatter import _formatting  # noqa: E402
```

Comment:
I think this can be moved outside of the `fixture`.

Reply:
I was not sure if having it import outside would lead to the "global" _Run be patched as well as the patched one, will give it a try and move it if it works

Reply:
OK, I was wrong, modified it